Makefile

```diff
@@ -209,8 +209,10 @@ validate: build-container
 # This target is only intended for development, e.g. executing it from an IDE. Use (make test) for CI or pre-release testing.
 test-all-local: validate-local test-unit-local
 
+.PHONY: validate-local
 validate-local:
 	hack/make.sh validate-git-marks validate-gofmt validate-lint validate-vet
+	hack/man-page-checker
```

Comment:
The `validate` target above should invoke `make validate-local` instead of open-coding it so that it also triggers this check.

Reply:
I don't feel comfortable making so many changes to this Makefile - there's a lot I don't understand, especially wrt `$(CONTAINER_RUN)`, and `hack/make.sh`, and `contrib/cirrus/runner.sh`, and what-gets-run-where in Cirrus and in devel environments. I was hoping to do the smallest possible change, for ease of future maintainability; and it sounds like I didn't get it right. I will take a fresh look tomorrow.

Reply:
To be fair if we are not running `make validate` in CI, probably no-one else is running it anywhere else :)